test/config/testDb.js

```diff
@@ -0,0 +1,108 @@
+const dotenv = require("dotenv");
+dotenv.config();
+
+const mongoose = require("mongoose");
+const Hel = require("../../models/Hel");
+
+let chai = require("chai"),
+  should = chai.should(),
+  expect = chai.expect;
+
+describe("Database Tests", function() {
+  //Before starting the test, create a sandboxed database connection
+  //Once a connection is established invoke done()
+  before(function(done) {
+    this.timeout(10000);
+    mongoose.connect(process.env.mongoURITest, {
```

Comment:
no modifications to .env right?

Reply:
Yes you will need to add the MongoURITest by copy pasting MongoURI and replacing ChronoFactorem-dev with anything other than ChronoFactorem-dev or ChronoFactorem-production